tools/firmware-utils/src/tplink-safeloader.c

```diff
@@ -784,6 +854,8 @@ int main(int argc, char *argv[]) {
 		info = &eap120_info;
 	else if (strcmp(board, "ARCHERC9") == 0)
 		info = &e9_info;
+	else if (strcmp(board, "TLWR1043NDV4_EU") == 0)
```

Comment:
I think it would make sense to drop the `_EU` suffix here and in the image Makefile for now, as there are no other versions of the TL-WR1043v4 so far. If TP-Link makes proper use of their SupportList format, we might be able to support multiple versions with a single image (and if they don't we can split it later).